File: extract.js
import fs from 'fs';
import pdf from 'pdf-parse/lib/pdf-parse.js';

let dataBuffer = fs.readFileSync('public/HACK ARENA ROUND 1 SHORTLISTED TEAMS.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(err => {
    console.error("Error reading PDF:", err);
});

File: extract.py
import PyPDF2

try:
    with open('public/HACK ARENA ROUND 1 SHORTLISTED TEAMS.pdf', 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ''
        for page in reader.pages:
            text += page.extract_text() + '\n'
        with open('extracted_teams.txt', 'w', encoding='utf-8') as outfile:
            outfile.write(text)
except Exception as e:
    print(f"Error: {e}")
